[PECO-237] Direct results support (#51)
databricks/databricks-sql-nodejs: CHANGELOG.md

## 0.1.x (Unreleased)

> - Added support for DirectResults - feature that reduces amount of requests to server for fetching
>   data and speed up data fetching

Reviewer: ```suggestion
- Added support for DirectResults, which speeds up data fetches by reducing the number of server roundtrips when possible.
```

Reviewer: I think long-term we're going to reformat our changelogs for the OSS connectors. This is just a wording change.
